resteasy-dependencies-bom/pom.xml

```diff
@@ -86,7 +86,7 @@
         <version.weld.api>3.0.SP3</version.weld.api>
         <version.weld>3.0.4.Final</version.weld>
         <version.json.patch>1.3</version.json.patch>
-        <version.io.reactiverse.reactive-contexts>1.0.0</version.io.reactiverse.reactive-contexts>
+        <version.io.smallrye.concurrency>1.0.0-SNAPSHOT</version.io.smallrye.concurrency>
```

Comment:
When do you expect smallrye.concurrency will go 1.0.0(.Final) ?

Reply:
I'm not sure, but it could be within a matter of a few months.

Reply:
Just released 1.0.1